src/compose.ts

```diff
@@ -1,8 +1,8 @@
-import { ResponsesType, APPLICATION_JSON } from "./response";
-import { Schemas } from "./schema";
-import { normalizePath, getSchemaName } from "./util";
-import { REF, parseObject, parseArray } from "./parse";
-import { isObject, isArray } from "./dataType";
+import {APPLICATION_JSON, ResponsesType} from "./response";
+import {Schemas} from "./schema";
+import {getSchemaName, normalizeName, normalizePath} from "./util";
+import {parseArray, parseObject, REF} from "./parse";
+import {isArray, isObject} from "./dataType";
```

Comment:
Please don't include format change into this PR. 
I will enable prettier formatting to this project when I have time.

Reply:
👍  formatting should be removed now